README.md

```diff
@@ -1,23 +1,59 @@
-# govwifi-admin
+# Govwifi Admin API
 
-## Purpose
+This is the admin platform, a website where organisations can create and manage their GovWifi installation within their organisation.
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
 
-This is the User Admin UI for the [GovWiFi][link_govwifi] project.
+## Overview
 
-## How to install and use
+The API contains files and test suites that describe and maintain the desired functionality of the admin platform. It is primarily a Ruby-on-Rails Project.
+
+The API uses some GovUK Services to implement some the desired functions, including:
+
+* [GOV.UK Registers][registers] as a resource for Government organisations to select their names from a predetermined list of registered organisations as they create their accounts.
+
+* [GOV.UK Notify][notify] to handle sending out situational emails to users.
+
+* [GOV.UK Zendesk][zendesk] to handle forms submitted by the user within the app.
+
```

Comment:
The application also provides the following data for the RADIUS configuration via an S3 bucket:
- IP addresses.
- RADIUS secret keys.